.github/workflows/deploy-staging.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Install near CLI
-        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/near-cli-rs/releases/download/v0.8.0/near-cli-rs-installer.sh | sh
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/near/near-cli-rs/releases/download/v0.11.1/near-cli-rs-installer.sh | sh
```

Comment:
value from current `cargo near new`

Reply:
this repo might have an auto-update of contents with fresh `cargo near new` with @create-pull-request github action or smth